packages/@okta/vuepress-site/docs/guides/submit-app/submission1-general/index.md

```diff
@@ -30,7 +30,7 @@ In the General Settings page, you need to fill in the basic information about yo
 
 ## Customer support
 
-* **Public support contact** &mdash; include a public point of contact (support email or phone number) for users who need assistance with your integration. You can also include a link to a FAQ or troubleshooting guide to help with common issues. This information is shared with the customers in the OIN catalog description.
+* **Support contacts** &mdash; include one or more public contact points for users who need assistance with your integration. You could also add a link to a FAQ or a troubleshooting guide. Use the drop-down menu to specify if you are adding an email, a URL, or a phone number and click **Add Another** to add additional contacts. Okta shares this information with customers in the OIN catalog description for your app integration.
```

Comment:
Suggest "can" instead of "could"; I pronounce FAQ as eff-a-q, so, would use "an", but quick search of dev docs shows "a faq" used elsewhere. Everything else looks good.

Reply:
Changes made - good suggestions, thanks!